### PR TITLE
feat: add exercise C The Odin Project

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md [hunk 1]
```diff
@@ -7,26 +7,100 @@ dashedName: css-foundations-exercise-c
 
 # --description--
 
-description
+Let's build a little off the previous exercise, in which you added multiple classes to a single element in order to apply two different rules to it.
+
+1. You should see a black background and white text on the first element.
+1. You should see a yellow background on the second element.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md:13]:
```suggestion
1. You should see a black background and white text on the first `button` element.
1. You should see a yellow background on the second `button` element.
```

It's not really clear what elements should be styled. `head` and `body` are also elements.

File: curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md [hunk 2]
```diff
@@ -35,10 +109,55 @@ Test 3
 ## --seed-contents--
 
 ```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Grouping Selectors</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md:116]:
```suggestion
    <link rel="stylesheet" href="styles.css">
```

File: curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md [hunk 3]
```diff
@@ -35,10 +109,55 @@ Test 3
...
 
 ```html
+<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grouping Selectors</title>
+    <link rel="stylesheet" href="solution.css">
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md:139]:
```suggestion
    <link rel="stylesheet" href="styles.css">
```

File: curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md [hunk 4]
```diff
@@ -7,26 +7,100 @@ dashedName: css-foundations-exercise-c
...
 ```js
+const classOne = document.querySelectorAll('button')?.[0].classList?.[0];
+const classTwo = document.querySelectorAll('button')?.[1].classList?.[0];
+
+function eitherOr() {
+  const a = new __helpers.CSSHelp(document)
+  return a.getStyle(`.${classOne}, .${classTwo}`) ?? a.getStyle(`.${classTwo}, .${classOne}`);
+}
+assert.equal(eitherOr()?.fontSize, '28px');
+assert.equal(eitherOr()?.fontFamily, 'Helvetica, "Times New Roman", sans-serif');
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-css-foundations-projects/css-foundations-exercise-c.md:80]:
```suggestion
assert.exists(eitherOr());
```

We should only test that the selector exists, here.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
A few nits.

**Review by naomi-lgbt — APPROVED** [review]
Okie dokie

**Review by Nirajn2311 — APPROVED** [review]

**Pull request merged**